### ISSUE TITLE
odoo 12 multisession POS freezes on Sync Orders 

### ISSUE THREAD

**Issue by alaherma** [itpp-labs/pos-addons#924]
Hi i installed odoo pos_multi_session module on odoo 12.
I already changed to "workers = 1"
and my sync server is " //localhost:8080"

but when loading the POS it just stays on Sync Orders  for ever

what am i missing?

**Comment by gustavovalverde** [comment]
Is the console showing an error? (F12 for developer tools)

**Comment by gustavovalverde** [comment]
@ilmir-k now that I see the instructions, it might be confusing that the syn server shown is `//localhost:8080` which won't work based on the nginx configuration on the same instructions. https://github.com/it-projects-llc/pos-addons/blob/11.0/pos_multi_session_sync/doc/index.rst

It would be nice to also explain to newcomers that the sync server has to be contacted from the local (POS) network, not from the Odoo (server) network.

**Comment by shabeervpk** [comment]
Hi @gustavovalverde, 
Am a newcomer, Can you please provide the details regarding Sync Server, the instruction are confusing for me.

1. is Sync server is an instance of Odoo?
2. Where to setup the Sync server : Can it be along with Odoo in cloud (same server) OR it should be in local?
3. In the documentation : [https://github.com/it-projects-llc/pos-addons/blob/11.0/pos_multi_session_sync/doc/index.rst](url)
Nginx conf of Sync server showing Odoo running on 8079 and 8082 (long-polling), But in main_server UI config of //localhost:8080
Is that correct?

All your POS modules are great!! I had setup the **POS in Odoo11, without any seperate sync server** (main_server **worker=2**, as report printing requires workers>1) with 1posbox from raspberry pi, am facing some issues:

- Sometimes Sync are conflicting, after that any of the kitchen/receipt printers are not printing.
is it required to have a separate sync server in this case?

Can you provide some details regarding the Sync Server

**Comment by ilmir-k** [comment]
@gustavovalverde 
Yes, right. Thanks for reporting this, I'll make the changes in the instruction

**Comment by keadanis** [comment]
Hello, I want to ask if there is any information or support to correctly configure the pos_multi_session_restaurant module.
I have followed the module guides, I have read some information that I have found on the internet, but I have not been able to make the module work in version 12.
I install and configure without problem, but when I open the point of sale I do not see the floors or tables configured.

Please help me a little bit about it. Thank you very much
